scripts/isct_test_function.py

```diff
@@ -433,6 +433,39 @@ def get_parser():
         print results_display.to_string()
         print 'Status: 0: Passed | 1: Crashed | 99: Failed | 200: Input file(s) missing | 201: Ground-truth file(s) missing'
 
+        if verbose == '2':
```

Comment:
Yes, it would make more sense, I just used the parameter as it was in the code. We should centralize these kinds of guidelines somewhere.